indexer/src/main/java/au/org/aodn/esindexer/configuration/DatasetAccessConfig.java

```diff
@@ -0,0 +1,17 @@
+package au.org.aodn.esindexer.configuration;
+
+import au.org.aodn.esindexer.service.DatasetAccessServiceImpl;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class DatasetAccessConfig {
+
+    @Bean(name = "DatasetAccessService")
```

Comment:
The repo is data-access-service, maybe you call it the same way, DataAccessService, be less varied in names.